paasta_tools/cleanup_kubernetes_cr.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python
+# Copyright 2015-2018 Yelp Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Usage: ./cleanup_kubernetes_cr.py [options]
+
+Command line options:
+
+- -d <SOA_DIR>, --soa-dir <SOA_DIR>: Specify a SOA config dir to read from
+- -v, --verbose: Verbose output
+"""
+import argparse
+import logging
+import os
+import sys
+from typing import Any
+from typing import Mapping
+from typing import NamedTuple
+from typing import Sequence
+
+import service_configuration_lib
+
+from paasta_tools.kubernetes_tools import delete_custom_resource
+from paasta_tools.kubernetes_tools import KubeClient
+from paasta_tools.kubernetes_tools import KubeKind
+from paasta_tools.kubernetes_tools import list_custom_resources
+from paasta_tools.utils import DEFAULT_SOA_DIR
+from paasta_tools.utils import load_system_paasta_config
+from paasta_tools.utils import SystemPaastaConfig
+
+log = logging.getLogger(__name__)
+
+
+class CustomResource(NamedTuple):
+    file_prefix: str
+    version: str
+    kube_kind: KubeKind
+    group: str
+
+
+def load_custom_resources(system_paasta_config: SystemPaastaConfig) -> Sequence[CustomResource]:
```

Comment:
same - this should probably be in a place it can be reused

Reply:
done